docker/pypi/wmagent-mariadb/Dockerfile

```diff
@@ -52,20 +48,14 @@ ADD my.cnf ${MDB_CONFIG_DIR}/my.cnf
 
 ENV PATH="/usr/local/bin/:${MDB_ROOT_DIR}:${PATH}"
 
-# set MariaDB docker specific bash prompt and manage alias for all users:
-RUN <<EOF cat >>/root/.bashrc
-alias manage=$MDB_MANAGE_DIR/manage
-export PS1="(MariaDB-$MDB_TAG) [\u@\h:\W]\$([[ \$(id -u) -eq 0 ]] && echo \# || echo \$) "
-EOF
-
-RUN <<EOF cat >>/home/cmst1/.bashrc
-alias manage=$MDB_MANAGE_DIR/manage
-export PS1="(MariaDB-$MDB_TAG) [\u@\h:\W]\$([[ \$(id -u) -eq 0 ]] && echo \# || echo \$) "
+# Set command prompt for root
+RUN <<EOF cat >> /root/.bashrc
+export PS1="(MariaDB-$MDB_TAG) [\u@\h:\W]# "
 EOF
 
-# RUN chown -R ${USER} ${MDB_ROOT_DIR}
+# allow dynamic users to create homefolders and .bashrc 
+RUN chmod 777 /home
```

Comment:
I am happy to hear about better alternatives to be able to create a home folder at run time.
Another alternative would be to define it at the host, and pass to the container via mount point, the only problem is that we actually have variations of bashrc (and manage).